pkgs/tools/security/jaeles/default.nix

```diff
@@ -5,23 +5,24 @@
 
 buildGoModule rec {
   pname = "jaeles";
-  version = "0.17";
+  version = "0.17.1";
 
   src = fetchFromGitHub {
     owner = "jaeles-project";
     repo = pname;
     rev = "beta-v${version}";
-    hash = "sha256-IGB+TYMOOO7fvRfDe9y+JSXuDSMDVJK+N4hS+kezG48=";
+    hash = "sha256-IGvIjO1nCilg2sPyScGTH5Zmv0rORlGwRv3NRxQk+aM=";
   };
 
-  vendorSha256 = "sha256-R2cP5zNuGUs0/KeaGhbQm1m5gVBVhpcFrS/jsph3EBk=";
+  vendorHash = "sha256-/Ow2qdcFduZ2ZyUUfCqpZxSh9yy3+tI/2N9Wl1fKXVI=";
 
   # Tests want to download signatures
   doCheck = false;
 
   meta = with lib; {
     description = "Tool for automated Web application testing";
     homepage = "https://github.com/jaeles-project/jaeles";
+    changelo = "https://github.com/jaeles-project/jaeles/releases/tag/beta-v${version}";
```

Comment:
```suggestion
    changelog = "https://github.com/jaeles-project/jaeles/releases/tag/${rev}";
```

Reply:
I prefer to stick to `${version}` instead of `${src.rev}`. I know that `nix-init` in the meantime suggests `${src.rev}` but changing `src` (happens from time to time for Python modules) forces one to update the changelog entry.

Typo fixed.